esphome/components/ili9xxx/ili9xxx_display.cpp

```diff
@@ -181,9 +185,28 @@ void ILI9XXXDisplay::display_() {
 
     while (rem > 0) {
       uint32_t sz = std::min(rem, ILI9XXX_TRANSFER_BUFFER_SIZE);
-      //  ESP_LOGVV(TAG, "Send to display(pos:%d, rem:%d, zs:%d)", pos, rem, sz);
+      ESP_LOGVV(TAG, "Send to display(pos:%d, rem:%d, zs:%d)", pos, rem, sz);
```

Comment:
```suggestion
      //  ESP_LOGVV(TAG, "Send to display(pos:%d, rem:%d, zs:%d)", pos, rem, sz);
```
This log message will slowdown the esp to much.